code/WorkInProgress/laundry.dm

```diff
@@ -58,14 +60,18 @@
 		src.generate_html()
 		return
 
-	if (src.cycle_current >= src.cycle_time) // cycle done!
+	var/mob/living/carbon/human/H = src.occupant
+	if ((!src.occupant && src.cycle_current >= src.cycle_time) || (src.occupant && src.cycle_current >= src.cycle_time_mob_inside)) // cycle done! The cycle is faster if a human is inside
 		if (src.cycle == WASH) // we have to dry things now!
 			for (var/obj/item/I in src.contents)
 				if (istype(I, /obj/item/clothing))
 					var/obj/item/clothing/C = I
 					C.stains = list("damp")
 					C.UpdateName()
 				I.clean_forensic()
+			if (src.occupant)
+				if (H.sims)
+					H.sims.affectMotive("Hygiene", 100)
```

Comment:
?.

Reply:
This was supposed to max out the hygiene motive on rp since its technically a washing. Did i put in the wrong one?

Reply:
no, use the `?.` operator

Reply:
Done, tell me if i messed it up